plugins/ocp/ocp-nvme.c

```diff
@@ -942,3 +945,65 @@ static int eol_plp_failure_mode(int argc, char **argv, struct command *cmd,
 
 	return err;
 }
+
+static int clear_pcie_corectable_error_counters(int argc, char **argv,
+						struct command *cmd,
+						struct plugin *plugin)
+{
+	const char *desc = "OCP Clear PCIe Correctable Error Counters";
+	__u32 result = 0;
+	__u32 clear_pcie_error_counters = 1 << 31;
+	struct nvme_dev *dev;
+	int uuid_index = 0;
+	bool no_uuid = false;
+	int err;
+
+	OPT_ARGS(opts) = {
+		OPT_FLAG("no-uuid", 'n', &no_uuid,
+			 "Skip UUID index search (UUID index not required for OCP 1.0)"),
+		OPT_END()
+	};
+
+	err = parse_and_open(&dev, argc, argv, desc, opts);
+	if (err)
+		return err;
+	if (no_uuid == false) {
```

Comment:
Personally I get easily confused by inverted logic. Wouldn't it make sense have a `bool uuid = true` as default?